clvm-utils/src/curried_program.rs

```diff
@@ -0,0 +1,67 @@
+use clvm_traits::{
+    clvm_list, clvm_quote, destructure_list, destructure_quote, match_list, match_quote, FromClvm,
+    MatchByte, Result, ToClvm,
+};
+use clvmr::{allocator::NodePtr, Allocator};
+
+#[derive(Debug, Clone)]
+pub struct CurriedProgram<T>(NodePtr, T);
```

Comment:
the members of this tuple needs to be public, don't they?
When used from another crate, I get an error because these are private fields

Reply:
Yep